src/search/views/Search.tsx

```diff
@@ -607,14 +574,23 @@ export const Search: FunctionComponent<SearchProps> = ({ history, location }: Se
 										<TextInput
 											id="query"
 											placeholder="Vul uw zoekterm in..."
-											value={formState.query}
+											value={searchTerms}
 											icon="search"
 											onChange={setSearchTerms}
 										/>
 									</FormGroup>
 									<FormGroup inlineMode="shrink">
 										<Button label="Zoeken" type="primary" onClick={copySearchTermsToFormState} />
 									</FormGroup>
+									<FormGroup inlineMode="shrink">
```

Comment:
What is the reasoning behind putting the hasFilters() around Button and not around the FormGroup? Or would that not mean "rendering less components"? 🤔

Reply:
fixed